index.html

```diff
@@ -1,3 +1,3 @@
 <!DOCTYPE html>
 <link rel="icon" href="data:;base64,iVBORw0KGgo="/>
-<meta http-equiv="refresh" content="0;URL=/v10/"/>
+<meta http-equiv="refresh" content="0;URL=/v11/"/>
```

Comment:
Can you change this back to v10 by hand?

Reply:
Fixed.